requirements-format.in

```diff
@@ -0,0 +1 @@
+black
```

Comment:
I think this actually makes sense as its own separate requirements file. `black` doesn't need any of the test or dev dependencies to be installed in order to work, and doesn't need to use the same version of Python as the test or dev venvs. Jenkins can run `make checkformatting` really fast as there are no deps to install and, if `checkformatting` fails, the whole Jenkins job will fail really fast